src/Akka.Persistence.SqlServer.Tests/DbUtils.cs

```diff
@@ -46,6 +46,11 @@ public static void Initialize(string connectionString)
                 connectionBuilder.InitialCatalog = databaseName;
                 ConnectionString = connectionBuilder.ToString();
             }
+
+            // Delete local snapshot flat file database
+            var path = "./snapshots";
```

Comment:
Did we have to do this because this was accidentally checked into source?

Reply:
In other words, why did this only happen upon upgrading to v1.4.19?

Reply:
I really have no idea, we always use the local persistent snapshot database and it never causes this problem.

Reply:
We reuse the same persistence ids for some of the tests, "a", "b", "c". It might be a transient thing, the order of test execution might have changed somehow, and the test that actually checks for sequence number now got executed after another test who doesn't care.

Reply:
Ah, these were batching sql journal tests that failed originally - does that make any difference?

Reply:
Notably, these were changes for v1.4.19 - not v1.4.20, which is where I believe we actually deployed all of your code changes...

Reply:
Might be? it could also have something to do with how XUnit does it test discovery or how msbuild does its code optimization

Reply:
@Aaronontheweb @Arkatufus this seems related to my comments here: https://github.com/akkadotnet/akka.net/pull/4892#discussion_r658530093

And in fact, deleting the snapshot folder was also the solution for the Oracle plugin, when most of the `ByTag` and `AllEvents` queries started failing after upgrading to 1.4.19.

I just hope we don't also have to delete old snapshots when we upgrade our system (we're currently still in 1.4.18).